packages/common/actions/entities/member.js

```diff
@@ -167,6 +175,31 @@ export const removeTeamMembersBatch = memberIds => ({
   },
 });
 
+export const grantAccessTeamMemberRequest = memberId => ({
+  type: GRANT_ACCESS_TEAM_MEMBER_REQUEST,
+  payload: {
+    memberId,
+  },
+});
+
+export const grantAccessTeamMemberSuccess = memberId => ({
```

Comment:
Do you really need this actions (success, failure and batch)? I don't see where it's used.